src/auth.rs

```diff
@@ -14,7 +14,7 @@ pub fn is_authorized(principal: &Principal, auth: Auth) -> bool {
 
 pub fn require_admin_or_controller() -> Result<(), String> {
     let caller = ic_cdk::caller();
-    if is_authorized(&caller, Auth::ManageService) || ic_cdk::api::is_controller(&caller) {
+    if is_authorized(&caller, Auth::ManageCanister) || ic_cdk::api::is_controller(&caller) {
```

Comment:
Renaming `ManageService` to `ManageCanister` for clarity (since in all other cases the term "service" is used to describe RPC services rather than the EVM RPC canister itself). Updated the design document accordingly.

Reply:
`ManageCanister` could be confused with the management canister, so this name is also not perfect...
What do you think?

Reply:
Right, I see what you're saying. Maybe just `Manage` then? I'll do this refactor in a separate PR so we can try a few different options.